cmd/dendrite/main.go

```diff
@@ -30,16 +31,33 @@ import (
 )
 
 var (
-	httpBindAddr  = flag.String("http-bind-address", ":8008", "The HTTP listening port for the server")
-	httpsBindAddr = flag.String("https-bind-address", ":8448", "The HTTPS listening port for the server")
-	certFile      = flag.String("tls-cert", "", "The PEM formatted X509 certificate to use for TLS")
-	keyFile       = flag.String("tls-key", "", "The PEM private key to use for TLS")
+	unixSocket           = flag.String("unix-socket", "", "The HTTP listening unix socket for the server")
+	unixSocketPermission = flag.Int("unix-socket-permission", 0755, "The HTTP listening unix socket permission for the server")
```

Comment:
Please mark these flags as experimental and not stable, as we will likely move these around over the coming months. Just put `EXPERIMENTAL(unstable): ` in each description line.

Reply:
fixed